tests/ci/codebuild/linux-x86/centos-7_gcc-4x.yml

```diff
@@ -9,7 +9,6 @@ phases:
       - if [ $(gcc -dumpversion) == 4.8.5 ]; then echo "Found correct gcc version 4.8.5"; else gcc --version && echo "gcc version mismatch" && exit 1; fi
       - export CC=gcc
       - export CXX=g++
```

Comment:
Can this be deleted and use tests/ci/codebuild/linux-x86/run_posix_tests.yml?

Reply:
Deleted.